model-server/src/main/kotlin/org/modelix/model/server/store/ObjectInRepository.kt

```diff
@@ -16,6 +16,7 @@
 
 package org.modelix.model.server.store
 
+// Order of fields must be consistent with the configuration of JdbcType in ignite.xml
```

Comment:
Let's just use this change to create a proper kdoc for this class?

Reply:
Done